featuretools/wrappers/sklearn.py

```diff
@@ -0,0 +1,261 @@
+        self.seed_features = seed_features
+        self.drop_contains = drop_contains
+        self.drop_exact = drop_exact
+        self.where_primitives = where_primitives
+        self.max_features = max_features
+        self.save_progress = save_progress
+        self.training_window = training_window
+        self.approximate = approximate
+        self.chunk_size = chunk_size
+        self.n_jobs = n_jobs
+        self.dask_kwargs = dask_kwargs
+        self.verbose = verbose
+        self.profile = profile
+
+    def fit(self, entity_ids, y=None):
+        """Wrapper for dfs
+
+            Calculates a feature matrix and features given a dictionary of entities and a list of relationships.
+
+            Args:
+                entity_ids (list): List of instances to calculate features on.
+
+            See Also:
+                :func:`synthesis.dfs`
+        """
+        self.feature_defs =    dfs(entities=self.entities,
+                                   relationships=self.relationships,
+                                   entityset=self.entityset,
+                                   target_entity=self.target_entity,
+                                   instance_ids=entity_ids,
+                                   agg_primitives=self.agg_primitives,
+                                   trans_primitives=self.trans_primitives,
+                                   allowed_paths=self.allowed_paths,
+                                   max_depth=self.max_depth,
+                                   ignore_entities=self.ignore_entities,
+                                   ignore_variables=self.ignore_variables,
+                                   seed_features=self.seed_features,
+                                   drop_contains=self.drop_contains,
+                                   drop_exact=self.drop_exact,
+                                   where_primitives=self.where_primitives,
+                                   max_features=self.max_features,
+                                   save_progress=self.save_progress,
+                                   features_only=True,
+                                   training_window=self.training_window,
+                                   approximate=self.approximate,
+                                   chunk_size=self.chunk_size,
+                                   n_jobs=self.n_jobs,
+                                   dask_kwargs=self.dask_kwargs,
+                                   verbose=self.verbose)
+        return self
+
+    def transform(self, entity_ids, y=None):
+        """Wrapper for calculate_feature_matix
+
+            Calculates a matrix for a given set of instance ids and calculation times.
+
+            Args:
+                entity_ids (list): List of instances to calculate features on.
+
+            See Also:
+                :func:`computational_backends.calculate_feature_matrix`
+        """
+        X_transformed =    calculate_feature_matrix(features=self.feature_defs,
```

Comment:
`calculate_feature_matrix` isn't guaranteed to return in the same order as the instance ids provided. you should add 

`return X_transformed.loc[entity_ids]` to ensure the order is correct to this method